src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,43 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const currentTab = tabs.find(({ id }) => id === selectedTabId) || tabs[0].id;
+  const onClickTab = (tab) => {
+    if (currentTab.id !== tab.id) {
+      onTabSelected(tab);
+    }
+  };
```

Comment:
```suggestion
  const handleTabClick = (tabId) => {
    if (currentTab.id !== tabId) {
      onTabSelected(tabId);
    }
  };
```
And pass to this handler just id to set it, and better to name it with `handler`

Reply:
how can we pass just the `id` if after that this `tabId` would go to `setSelectedTab` ? Since we removed `[selectedTabId, setSelectedTabId]` it wouldnt work and we need to 'transfer' whole element